science/digital_rf/Portfile

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+PortGroup           cxx11 1.1
+PortGroup           python 1.0
+
+github.setup        MITHaystack digital_rf 2.6.0
+maintainers         @MITHaystack {mit.edu:rvolz @ryanvolz} {mit.edu:swoboj @jswoboda}
+description         Read, write, and interact with data in the Digital RF and Digital Metadata formats.
+long_description    Tools to store RF and metadata for scientific engineering research projects.
```

Comment:
The long description is scarcely different from the short one here, and is actually shorter. I'd rewrite it to actually explain what the content is. The purpose of the long description is to help someone figure out if they are interested in the software.

Reply:
I will update it to the following: The Digital RF project encompasses a standardized HDF5 format for reading and writing of radio frequency data and the software for doing so. The format is designed to be self-documenting for data archive and to allow rapid random access for data processing. For details on the format, refer to the 'documents' directory in the source tree.

Reply:
Sounds good.